docs/management_api.md

```diff
@@ -30,7 +30,7 @@ Similar to the [Inference API](inference_api.md), the Management API provides a
 * response_timeout - If the model's backend worker doesn't respond with inference response within this timeout period, the worker will be deemed unresponsive and rebooted. The units is seconds. The default value is 120 seconds.
 
 ```bash
-curl -X POST "http://localhost:8081/models?url=https://<s3_path>/squeezenet_v1.1.mar"
+curl -X POST "http://localhost:8081/models?url=https://<cloud_OR_noncloud_storage>/squeezenet_v1.1.mar"
```

Comment:
Replace cloud_OR_noncloud_storage with links for squeezenet here https://github.com/pytorch/serve/blob/master/docs/model_zoo.md